node/node.go

```diff
@@ -869,8 +874,10 @@ func NewNodeWithContext(ctx context.Context,
 	}
 
 	// Determine whether we should do fast sync. This must happen after the handshake, since the
-	// app may modify the validator set, specifying ourself as the only validator.
-	fastSync := config.FastSyncMode && !onlyValidatorIsUs(state, pubKey)
+	// app may modify the validator set, specifying ourselves as the only validator.
+	fastSync := config.FastSyncMode && !onlyValidatorIsUs(state,
+		pubKey) // the state certainly has the latest timeouts according
+	// the app version lastBlock in the blockstore
```

Comment:
[blocking] this comment looks incomplete and not part of the previous comment. If it's meant to be combined, can this move the multi-line comment _above_ the line that it is applicable to?

Reply:
This comment wasn't addressed.